### PR TITLE
[SIE-51] and [SIE-50] Photo Uploads

### PR DESCRIPTION
**Pull request by alecherryy**
This PR includes work for [SIE-50](https://cs6510.atlassian.net/browse/SIE-50) and [SIE-51](https://cs6510.atlassian.net/browse/SIE-51). With this work, we can:
1. toggle a device's webcam ON/OFF
2. take a photo
3. show a preview of the photo

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
     "@storybook/react": "^6.1.17",
     "eslint-config-google": "^0.14.0",
     "react-router-dom": "^5.2.0",
-    "react-step-wizard": "^5.3.5"
+    "react-step-wizard": "^5.3.5",
+    "react-webcam": "^5.2.3"
```
Comment (alecherryy) [inline package.json:54]: New package to enable webcam.

File: src/scss/_mixins.scss [hunk 2]
```diff
@@ -49,6 +49,30 @@
   }
 }
 
+// display styled ordered list
+@mixin list-styled-ordered-list {
```
Comment (alecherryy) [inline src/scss/_mixins.scss:53]: New `@mixin` for fancy ordered lists.

File: src/stories/components/WebcamControls/WebcamControls.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+  // toggle device camera
+  const toggleCamera = () => {
+    setWebcamOn(!cameraIsOn);
+  };
+
+  // take a photo
+  const capturePhoto = React.useCallback(
+    () => {
+      setSnapshot(webcamRef.current.getScreenshot());
```
Comment (venjason) [inline src/stories/components/WebcamControls/WebcamControls.js:32]: I am surprised at how quickly we are able to capture the image and present it! That's awesome :D.

**Review by venjason — APPROVED** [review]
Every thing looks good on my end! Was able to successfully run the webcam controls / take a picture with my web browser (chrome & safari).

**Pull request merged**